video_xblock/static/html/wistiavideo.html

```diff
@@ -1,2 +1,2 @@
-<video id="vid1" muted="muted" src="" class="video-js vjs-default-skin" preload="auto" width="640" height="360" data-setup='{ "techOrder": ["wistia"], "sources": [{ "type": "video/wistia", "src": "{{ url }}" }] }'>
+<video id="vid1" smuted="muted" src="" class="video-js vjs-default-skin" controls width="640" height="264" data-setup='{ "techOrder": ["wistia"], "sources": [{ "type": "video/wistia", "src": "{{ url }}?controlsVisibleOnLoad=false" }], "playbackRates": [0.5, 1, 1.5, 2] }'>
```

Comment:
smuted?
Please put each attribute on a separate line like we have in the youtube.html